tests/variables.tf

```diff
@@ -0,0 +1,50 @@
+variable "build_id" {
+  default = ""
+  type    = "string"
+}
+
+variable "cloudwatch_agent_url_lx" {
+  default = "s3://amazoncloudwatch-agent/linux/amd64/latest/AmazonCloudWatchAgent.zip"
+  type    = "string"
+}
+
+variable "cloudwatch_agent_url_win" {
+  default = "s3://amazoncloudwatch-agent/windows/amd64/latest/AmazonCloudWatchAgent.zip"
+  type    = "string"
+}
+
+variable "instance_role" {
+  default = ""
+  type    = "string"
+}
+
+variable "keypair_name" {
+  default = ""
+  type    = "string"
+}
+
+variable "private_subnet" {
```

Comment:
Not sure I understand the purpose of this. This is only used in the rules for the security group, but the only thing that should _need_ access is the build system (codebuild), which is handled already through the `data.http.ip` resource.

Any other extra-special access for those extra-special circumstances should be handled just fine through the `security_group` option.

Reply:
Ingress rules removed from the sg per offline conversation.